grpc/pb-marshalling.go

```diff
@@ -323,14 +323,13 @@ func PBToAuthz(pb *corepb.Authorization) (core.Authorization, error) {
 }
 
 func registrationValid(reg *corepb.Registration) bool {
-	return !(reg.Id == nil || reg.Key == nil || reg.Agreement == nil || reg.InitialIP == nil || reg.CreatedAt == nil || reg.Status == nil || reg.ContactsPresent == nil)
```

Comment:
Id: because new Registrations don't have an Id before they're written to the database (also tagged with 'omitempty' in objects.go)
Agreement: because objects.go tags it with 'omitempty'
Status: because new Registrations don't have a Status unless they're Deactivated
ContactsPresent: because it is a boolean, and false is a zero value